tagging/src/main/scala/com/softwaremill/tagging/TaggingCompat.scala

```diff
@@ -0,0 +1,13 @@
+package com.softwaremill.tagging
+
+import scala.language.higherKinds
+import scala.language.implicitConversions
+
+trait TaggingCompat {
```

Comment:
Maybe, as you already mentioned, we could make this generic with `Typeclass`? Then, you could enable tag lifting selectively for chosen typeclasses, e.g. so:

```
object Json4sTaggingCompat extends TaggingCompat[Serializer]
import Json4sTaggingCompat._
```

We could also have a `AnyTypeclassTaggingCompat` with the current ultra-generic definition. Wdyt?